src/main/scala/nl.knaw.dans.easy.bagstore/server/StoresServletComponent.scala

```diff
@@ -39,10 +39,22 @@ trait StoresServletComponent extends DebugEnhancedLogging {
 
     get("/") {
       contentType = "text/plain"
-      bagStores.storeShortnames
+      Ok(bagStores.storeShortnames
         .keys
         .map(store => s"<${ externalBaseUri.resolve(s"stores/$store") }>")
-        .mkString("\n")
+        .mkString("\n"))
+    }
+
+    get("/:bagstore") {
+      contentType = "text/plain"
+      val bagstore = params("bagstore")
+      bagStores.getBaseDirByShortname(bagstore)
+        .map(baseDir =>
```

Comment:
shouldn't this `baseDir` be used somewhere?

Reply:
No, actually we are only checking that it exists; but I will make it a blank parameter.